analyzer.go

```diff
@@ -190,6 +190,8 @@ func (f *AnalyzerFactory) setRun(analyzer *Analyzer, imageRef string) error {
 
 // Analyze fetches the layers metadata from the previous image and writes analyzed.toml.
 func (a *Analyzer) Analyze() (files.Analyzed, error) {
+	ft := log.NewFuncTimer("Analyzer", a.Logger)
```

Comment:
I think the 2 step approach of creating the timer and manually deferring is good to have, but maybe a convenience method would be nice for the most common approach?
```suggestion
	defer log.MeasureDuration("Analyzer", a.Logger)
```

Reply:
Thanks @jabrown85  - The `New` function is actually also writing a Start timestamp to the logs, so maybe I should call it `NewAndLog` or similar?   There is a 3-step version available if you (1) make the struct yourself, (2) call `RecordStart` and then do  ` defer RecordEnd()`

Reply:
or in other words,  I'm not sure there's a one-liner that outputs both a start and end log timestamp

Reply:
Ah, right. I think a method could be written like this

```
func MeasureDuration(msg string) {
  ft := log.NewRecordStart(msg, b.Logger)
  return ft.RecordEnd()
}

// caller
defer log.MeasureDuration("Analyzer", a.Logger)() <-- notice second function is deferred
```

Reply:
indeed it can! :)